### ISSUE TITLE
updater conflict

### ISSUE THREAD

**Issue by xLOR3x** [Shawminator/DayZeEditor#8]
There's been an issue with current update via launching the main app which, after some downloading process, was trying to start any app called "updater.exe" in the system, not just the one needed.
In my case DayZeEditor was unsuccessfully trying to update through RAGE:MP updater.exe app, located in completely different directory, making automatic updating impossible.

**Comment by Shawminator** [comment]
Not sure why as the updater.exe should be in the same directory as the dayZeEditor.exe.
can you check the updater.exe is there?

**Comment by xLOR3x** [comment]
Yep, it was there.

So, there were two updater.exe in the system, one was in, for example, c:/ragemp folder, another one in c:/dayzeeditor folder.
I've launched DayZeEditor, cmd showed some info about downloading an update and after that it automatically tried to launch updater.exe. Seems like it found the "wrong" one from another app and tried to launch it.
My guess would be that it was searching the system for "updater.exe" app, regardless of it's location or relation to the right one app.

There also was a warning from RAGE:MP's updater saying something like "c:/ragemp/updater.exe is trying to launch through unexpected app c:/dayzeeditor/dayzeeditor.exe". Unfortunately I have no screenshots of that warning and I already downloaded fresh DayZeEditor release.

**Comment by Shawminator** [comment]
Are you in the discord? If so shoot me a dm and we can test something out.

**Comment by Shawminator** [comment]
Pushed an Update but you will need to do it manaully this time agagin.
when the update fails make sure the dayz e editor is closed and just drag and drop the latest zip file to the updater.exe this will run the update program and extract all the files from the zip file.

**Comment by xLOR3x** [comment]
Sorry for ghosting you, completely forgot about this issue, rarely visit github.
Also couldn't find your discord.

The method you've described worked like a charm, everything's updated and working right.
First it went into failing just like previous time, then I drag&dropped 0.7.9.2 archive on DayZeEditor's updater.exe, console ran all the files and it launched an updated app.

Also I was able to screenshot the issue this time, maybe you'll be able to grab some info from there, I'm attaching it to this message.

![dayzeeditor updater](https://github.com/Shawminator/DayZeEditor/assets/160201550/3bdccc64-9cff-4bac-945d-a02a75759b17)
P.S. I censored the path of DayZeEditor, but anyways it's located in it's own folder on the desktop. Maybe this info will also help somehow.
